### PR TITLE
Modules/Developer-Edition: New - Dev version for Pro.

### PR DESCRIPTION
**Pull request by Nevoss**
ref: ED-600

### CODE REVIEW

File: bootstrap.php [hunk 1]
```diff
@@ -9,6 +9,7 @@
```
Comment (matipojo) [inline bootstrap.php:9]: Fix according to the rename to `elementor-beta`.

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — APPROVED** [review]

File: modules/developer-edition/version-control.php [hunk 2]
```diff
@@ -61,7 +61,7 @@ public static function on_activate_and_deactivate_plugin() {
 	 * @return object
 	 */
 	public function pre_set_site_transient_update_plugins( $transient ) {
-		$current_version = $this->get_elementor_plugin_data()['Version'];
+		$current_version = $this->get_plugin_version( Bootstrap::ELEMENTOR_PLUGIN_NAME );
```
Comment (matipojo) [inline modules/developer-edition/version-control.php:64]: Adapt to work with the pro plugin. (add a filter to `get_wp_org_data` in order to allow the pro plugin to set its  stable version)

**Review by matipojo — CHANGES_REQUESTED** [review]

File: tests/phpunit/unit/modules/developer-edition/test-pro-version-control.php [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+				],
+			],
+			'https://test.com'
+		);
+
+		// Assert
+		$this->assertArrayNotHasKey( 'dev', $result['body'] );
+	}
+
+	public function test_add_dev_param_to_get_version_request__when_elementor_pro_is_installed() {
```
Comment (matipojo) [inline tests/phpunit/unit/modules/developer-edition/test-pro-version-control.php:77]:
```suggestion
	public function test_add_dev_param_to_get_version_request__when_elementor_pro_is_not_installed() {
```

**Review by matipojo — CHANGES_REQUESTED** [review]

File: tests/phpunit/unit/mocks/api.php [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Mock for elementor pro api class.
+ */
+class API {
+	const STORE_URL = 'https://api.test.com';
```
Comment (matipojo) [inline tests/phpunit/unit/mocks/api.php:12]:
```suggestion
	const STORE_URL = 'https://api.test.local';
```

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — APPROVED** [review]

**Review by KingYes — APPROVED** [review]

**Pull request merged**